model/job/src/main/proto/beam_artifact_api.proto

```diff
@@ -23,9 +23,9 @@
 
 syntax = "proto3";
 
-package org.apache.beam.runner_api.v1;
+package org.apache.beam.model.job.v1;
 
-option java_package = "org.apache.beam.sdk.common.runner.v1";
+option java_package = "org.apache.beam.model.job.v1";
```

Comment:
Artifact should strictly speaking be split in 3 now: data structures, stage (submission) and retrieval (execution).